utils/earthquake_data.py

```diff
@@ -156,6 +156,13 @@ def get_weight_matrix(self, x_axis_name, y_axis_name, nbins_x, nbins_y):
         ybins = pd.Series(ybins, name=y_axis_name)
         return z, xbins, ybins
 
+    def get_default_timedelta(self):
+        """Return the default timedelta for each catalog.
+        """
+        if self.catalog_type == CatalogTypes.DAT_EXT:
+            return timedelta(weeks=4)
```

Comment:
Instead of using an `if` statement here, you could just return the default one week, and then override the function in the `BaselEarthquakeData` class.